packages/cli/test/functional/test_site/testPopovers.md

```diff
@@ -0,0 +1,67 @@
+**Popover with attributes**
```

Comment:
We can add more tests on:
- ensuring the priority scheme discussed is followed
- ensuring that if `src` is an url it will error
- and other tests that you have come up with